app/hub/Views/UpgradePlanView/UpgradePlanView.jsx

```diff
@@ -166,20 +166,14 @@ const UpgradePlanView = (props) => {
 		mobileComparisonTableRef.current.scrollIntoView({ behavior: 'smooth' });
 	};
 
-	// UTM params
-	const signedIn = +!!user;
-	const subscriptionType = () => {
-		if (isPremium) return 'PREMIUM';
-		if (isPlus) return 'SUPPORTER';
-		return '-1';
-	};
-	// Interval is the query Param to show monthly/yearly pricing in checkout web, also used as a ping parameter
+	// UTM and Query Params
+	// Subscription_interval is the query param to show monthly/yearly pricing in checkout web
 	const interval = show_yearly_prices ? 'year' : 'month';
-	const utmParams = `utm_source=gbe&signedIn=${signedIn}&st=${subscriptionType()}&subscription_interval=${interval}`;
+	const params = `utm_source=gbe&interval=${interval}`;
 
 	const plusCTAButton = (position) => {
 		const utm_campaign = position === 'top' ? 'c_1' : 'c_2';
```

Comment:
Could you throw a set of parens around the rvalue to make this a little easier to parse visually?

Reply:
Yes, yes I can :D